app/controllers/AuthenticationController.scala

```diff
@@ -621,6 +621,8 @@ class AuthenticationController @Inject()(
                     dataStoreToken <- bearerTokenAuthenticatorService.createAndInitDataStoreTokenForUser(user)
                     _ <- organizationService
                       .createOrganizationDirectory(organization._id, dataStoreToken) ?~> "organization.folderCreation.failed"
+                    _ <- Fox.runOptional(signUpData.acceptedTermsOfService)(version =>
+                      acceptTermsOfServiceForUser(user, version))
```

Comment:
@frcroth I would prefer if the sign up request gets rejected in case `acceptedTermsOfService` is not set to true or empty but the application conf says that TOS is enabled.

What do you think?